scripts/check_e2e_tests_are_captured_in_ci.py

```diff
@@ -42,7 +42,7 @@
 SAMPLE_TEST_SUITE_THAT_IS_KNOWN_TO_EXIST = 'coreEditorAndPlayerFeatures'
 
 
-def get_e2e_suite_names_from_jobs_travis_yml_file():
+def get_e2e_suite_names_from_suites_travis_yml_file():
```

Comment:
These are not going to be the suites name, right? (not sure!)

Reply:
Also, what does `from_suites` signify here? (I don't have any suggestion for the name, I think we should remove this part from the test completely!(not sure))

Reply:
Actually the `..._from_jobs` refers to the associated section in the Travis config, so I reverted the name.